### PR TITLE
Additions to sub nav for Automation Analytics:

### PR DESCRIPTION
**Pull request by kialam**
- Added new route for Job Explorer.
- Title Cased sub nav items.
- Re-ordered items within sub nav.

### CODE REVIEW

**Review by benthomasson — APPROVED** [review]
LGTM

File: main.yml [hunk 1]
```diff
@@ -431,7 +433,7 @@ openshift:
       - id: overview
         title: Overview
         group: openshift
-  source_repo: 
+  source_repo:
```
Comment (kruai) [inline main.yml:436]: What's up with this change?

Comment (kruai) [reply main.yml:436]: I mean it's probably fine though tbh

**Review by kruai — APPROVED** [review]

File: main.yml (hunk 1 above)
Comment (kialam) [reply main.yml:436]: Ah, probably my linter got it by accident. But yeah should be fine 👍

**Pull request merged**